### PR TITLE
Ml il8n the specs

### PR DESCRIPTION
**Pull request by marklohr**
https://www.pivotaltracker.com/story/show/157261689

If an institution wants to use SPARC in a language other than English, Feature specs will not pass because we have hard-coded English statements. Any time a spec interacts with an element on the page, it needs to use the i18n.t(...) format, rather than the text as a string to ensure it is safe for internationalization.

Task: Find all instances where text is hard-coded in feature specs and replace them with the corresponding i18n sequences.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ en:
     reply: "Reply"
     reset: "Reset"
     save: "Save"
+    save_continue: "Save and Continue"
```
Comment (kyle-glick) [inline config/locales/en.yml]: This value is pulled from `t(:proper)[:navigation][:bottom][:save_and_continue]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/navigation/_footer.html.haml

File: config/locales/en.yml [hunk 2]
```diff
@@ -461,6 +462,7 @@ en:
   protocols:
     new: "New Protocol"
     edit: "Edit %{protocol_type} Information"
+    edit_study: "Edit Study Information"
```
Comment (kyle-glick) [inline config/locales/en.yml]: This should be using the `edit` key above and passed a `protocol_type` parameter

File: config/locales/dashboard.en.yml [hunk 3]
```diff
@@ -223,6 +224,9 @@ en:
       documents:
         header: "Documents"
         add_document: "Add Document"
+        view_study_details: "View Study Details"
```
Comment (kyle-glick) [inline config/locales/dashboard.en.yml]: This value is pulled from `t('protocols.view_details.button')` in the application and needs a `protocol_type` parameter.

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/protocols/_summary.html.haml
https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/dashboard/protocols/_summary.html.haml

File: config/locales/proper.en.yml [hunk 4]
```diff
@@ -242,9 +254,11 @@ en:
         get_cost_estimate: "Get a Cost Estimate"
         save_and_continue: "Save and Continue →"
         submit: "Submit Request →"
+        submit_request: "Submit Request"
         saving: "Saving..."
         submitting: "Submitting..."
         getting_cost: "Getting Cost..."
+        view_study_details: "View Study Details"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from `t('protocols.view_details.button')` in the application and needs a `protocol_type` parameter.

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/protocols/_summary.html.haml
https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/dashboard/protocols/_summary.html.haml

File: config/locales/proper.en.yml [hunk 5]
```diff
@@ -242,9 +254,11 @@ en:
         get_cost_estimate: "Get a Cost Estimate"
         save_and_continue: "Save and Continue →"
         submit: "Submit Request →"
+        submit_request: "Submit Request"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from the above `submit` key

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/navigation/_footer.html.haml

File: config/locales/proper.en.yml [hunk 6]
```diff
@@ -53,6 +53,13 @@ en:
       accordion:
         header: "Browse Service Catalog"
       back_to_top: "Back to Top"
+      short_interaction: "Short Interaction"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from `t(:proper)[:right_navigation][:short_interaction][:header]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/right_navigation/_right_navigation.html.haml

File: config/locales/proper.en.yml [hunk 7]
```diff
@@ -53,6 +53,13 @@ en:
...
+      submit: "Submit"
+      add: "Add"
+      continue: "Continue"
+      contact: "Contact Us"
+      help: "Help/FAQs"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from `t(:proper)[:right_navigation][:faqs][:header]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/helpers/service_requests_helper.rb

File: config/locales/proper.en.yml [hunk 8]
```diff
@@ -53,6 +53,13 @@ en:
       accordion:
         header: "Browse Service Catalog"
       back_to_top: "Back to Top"
+      short_interaction: "Short Interaction"
+      submit: "Submit"
+      add: "Add"
+      continue: "Continue"
+      contact: "Contact Us"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from `t(:proper)[:right_navigation][:contact][:header]`

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/right_navigation/_right_navigation.html.haml

File: config/locales/proper.en.yml [hunk 9]
```diff
@@ -81,6 +88,8 @@ en:
     # PROTOCOL #
     ############
     protocol:
+      new_research_study: "New Research Study"
```
Comment (kyle-glick) [inline config/locales/proper.en.yml]: This value is pulled from `t(:protocols)[:studies][:new]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/protocol/_protocol_display.html.haml

**Review by kayla-glick — CHANGES_REQUESTED** [review]
This story shouldn't be adding any new YAML unless there are raw strings being used in the application that need to be YAMLized. I pointed out several cases where YAML was added but should instead be referencing an existing key based on how it's used in the application, but did not highlight all of them as it would take too long to do so.

**Comment by amcates** [comment]
@jleonardw9 @Stuart-Johnson I believe this pull request should just be trashed.  It's really old and I think we have moved past this.  Thoughts?

**Comment by Stuart-Johnson** [comment]
@amcates Agreed

**Pull request closed without merging**